### ISSUE TITLE
Download Videos fails

### ISSUE THREAD

**Issue by Sprocket02** [bropat/eufy-security-ws#176]
Basically I can connect to the station and all devices. Now I am trying to download videos. Here comes an error message that I unfortunately do not understand.

Command:

`{"command": "device.start_download", "messageId": "device.start_download" , "serialNumber": "T8161P102236XXXX", "path": "/etc/openhab/html/camera/"}`

Reply:

`"{"type":"event","event":{"source":"device","event":"command result","serialNumber":"T8161P102236XXXX","command":"start_download","returnCode":-104,"returnCodeName":"ERROR_INVALID_ACCOUNT","customData":{"command":{"name":"deviceStartDownload","value":{"path":"/etc/openhab/html/camera/"}}}}}"`

I also don't understand the meaning of the property `cipherId`. What do I enter there?

Basically I would like to use the preview image and the corresponding video in Openhab for each event of the camera.

Maybe someone can give me a short tip or share his code?

**Comment by PhilippEngler** [comment]
At the moment, this is not possible on the HomeBase 3:
[https://github.com/bropat/eufy-security-client/blob/7fc2a45a75ee1bf69ced36d351d1ec6d623896f6/src/http/station.ts#L3581](url)

**Comment by bropat** [comment]
It's like @PhilippEngler has already written.
This functionality must first be implemented in `eufy-security-client`.
